.semaphore/semaphore.yml

```diff
@@ -13,7 +13,6 @@ blocks:
       prologue:
         commands:
           - checkout
-          - cache restore m2
```

Comment:
I will restore this once we clear the cache, right now the checkstyle config for pmd is cached and is causing problems.

Reply:
Evidence of cached config: https://github.com/checkstyle/checkstyle/pull/9671 : https://checkstyle.semaphoreci.com/jobs/e514179f-c391-4886-96c6-c27c3c36aae4